src/components/Dashboard/Modals/EditAppModal/Tabs/GeneralTab/GeneralTab.tsx

```diff
@@ -44,6 +43,13 @@ export const GeneralTab = ({ form, openTab }: GeneralTabProps) => {
         variant="default"
         {...form.getInputProps('behaviour.externalUrl')}
       />
+
+      {!form.values.behaviour.externalUrl.startsWith('https://') &&
```

Comment:
Maybe use an array instead of two checks

Reply:
I don't think it changes anything, you can accept the PR and leave this as comment